### PR TITLE
Map cell value on uri

### PR DESCRIPTION
**Pull request by shashankmc**
Resolved the issue on #28 . Needs validation @matthijssloep

### CODE REVIEW

**Comment by aiaragomes** [comment]
APOE is not appearing in the publish preview for me.

**Review by aiaragomes — CHANGES_REQUESTED** [review]
There is still a problem with APOE, see comments.

File: management_webpage/flaskr/getCategories.py [hunk 1]
```diff
@@ -28,11 +28,13 @@ def getClassCategories(value):
       { ?category rdfs:label ?categoryLabel. }
       FILTER ( ?category != ?targetClass ).
     }'''%value
+    print(f"This query is being used  for obtaing \
```
[the remark this answers is not in the document]
Comment (shashankmc) [reply management_webpage/flaskr/getCategories.py]: Sorted

File: management_webpage/flaskr/mapDatasets.py [hunk 2]
```diff
@@ -179,29 +180,40 @@ def cellMapping():
...
+                # on delete cell link as well. So this method is changed as well.
+                useLinkdata.deleteCellLinks(selectedValue['categoryUri'].loc[0],\
+                        olderValue['categoryUri'].loc[0], cdmValue, catValue)
             else:
                 # Create new link
-                useLinkdata.createCellLink(baseUri, selectedValue['category'].loc[0], cdmValue, catValue)
+                print("No existing link detected, creating new ones. Yay.")
+                # As getCategoryCoede method has been changed, the effect is on
+                # create cell link as well, so this method is modified as well.
+                useLinkdata.createCellLink(selectedValue['categoryUri'].loc[0], cdmValue, catValue)
```
[the remark this answers is not in the document]
Comment (shashankmc) [reply management_webpage/flaskr/mapDatasets.py:216]: Resolved with new commit. New issue with "UNKNOWN" selection - if previous value was something meaningful and the selection is modified to "UNKNOWN", nothing happens. Need to bake in that condition.

**Comment by shashankmc** [comment]
@aiaragomes Should properly work now for unknown selection during cell mapping.

**Comment by shashankmc** [comment]
> APOE is not appearing in the publish preview for me.

@aiaragomes The header doesn't display "APOE" during preview but displays "ncbi:gene/348" is which the URI for APOE. Does that make sense?

**Pull request merged**